workflow.qmd

```diff
@@ -0,0 +1,41 @@
+---
+title: "U.S. Greenhouse Gas Center: Data Workflow Documentation"
+---
+
+Welcome to the [U.S. Greenhouse Gas (GHG) Center](https://earth.gov/ghgcenter) data workflow documentation. View data flow diagrams to discover the journey of each dataset from acquisition into the US GHG Center.
```

Comment:
Welcome to the homepage for [U.S. Greenhouse Gas (GHG) Center](https://earth.gov/ghgcenter) data flow diagrams.

Reply:
Change second sentence to: "Use these diagrams to discover the journey of each dataset from acquisition to integration in the US GHG Center."